packages/babel-generator/src/source-map.js

```diff
@@ -18,16 +18,16 @@ export default class SourceMap {
 
   get() {
     if (!this._cachedMap) {
-      const map = this._cachedMap = new sourceMap.SourceMapGenerator({
+      const map = (this._cachedMap = new sourceMap.SourceMapGenerator({
```

Comment:
Like that `return` above, interesting. Not sure why you'd want parens around this.